### ISSUE TITLE
app crashed - waiting for file changes before starting

### ISSUE THREAD

**Issue by TheoMer** [lvarayut/relay-fullstack#31]
OS: Windows 10 Pro
NPM: 3.9.5

So I:
1. git clone https://github.com/lvarayut/relay-fullstack.git
2. cd relay-fullstack
3. npm install
4. npm start

and am then issued with the following message:

`> relay-fullstack@1.0.0 start C:\Users\Theo\Documents\Visual Studio 2015\Projects\relay-fullstack`
`> nodemon --watch 'server/data/**/*.js' --exec 'npm run update && babel-node server/index.js'`

`[nodemon] 1.10.0`
`[nodemon] to restart at any time, enter`rs`
`[nodemon] watching: 'server/data/**/*.js'`
`[nodemon] starting `'npm run update`
`''npm' is not recognized as an internal or external command,`
`operable program or batch file.`
`[nodemon] app crashed - waiting for file changes before starting...`

What is the issue here?

**Comment by lvarayut** [comment]
Thanks for the question. The issue seems to be related to your environment setup. Please take a look at this [Stack Overflow question](http://stackoverflow.com/questions/20992723/npm-is-not-recognized-as-internal-or-external-command-operable-program-or-bat).

**Comment by TheoMer** [comment]
@lvarayut Yes, I have C:\Program Files\nodejs as part of my PATH variable, but that does not resolve the issue. (See image)

I have 4 other projects, all using 'npm start', which all work with no issues. Yours is the only one that fails when running that commenad.

![npm path variable](https://cloud.githubusercontent.com/assets/15969262/17648152/f2869e18-6203-11e6-98e4-a25533e09e69.jpg)

If I run `npm run update` I get `Schema has been regenerated`. But if I run `npm start`, the same error, as mentioned above, occurs. So, why would `npm run update` work but not `npm start`?

**Comment by lvarayut** [comment]
Would [this issue](https://github.com/lvarayut/relay-fullstack/issues/16) be related to your problem?

**Comment by TheoMer** [comment]
@lvarayut So the solution was to change:

`"start": "nodemon --watch 'server/data/**/*.js' --exec 'npm run update && babel-node server/index.js'"`

to 

`"start": "nodemon --watch server/data/**/*.js --exec \"npm run update && babel-node server/index.js\""`

You may wish to update the 'Read me' section to notify windows users of this solution, should they experience this issue.

**Comment by lvarayut** [comment]
Thanks for sharing the solution. I will find the best way to solve the problem.

**Comment by reinert** [comment]
@TheoMer THANK YOU! Saved my day!

**Comment by gvortel** [comment]
@TheoMer you are the man! Thank u

**Comment by LucasMatuszewski** [comment]
I had a similar Nodemon error for different script.
Changing ' to \" didn't helped me. But I found other solution.
Maybe it will help somebody:

Error:
```
[nodemon] starting `node build/bundle.js`
'\"node build\bundle.js\"' is not recognized as an internal or external command,
operable program or batch file.
[nodemon] app crashed - waiting for file changes before starting...
```

I fixed it by changing package.json script from:

`"dev:server": "nodemon --watch build --exec \"node build/bundle.js\"",`

to:

`"dev:server": "nodemon build/bundle.js --watch build",`

works fine for me.

Best regards :)
Lukas
